docs/fsharp-cheatsheet.md

```diff
@@ -125,33 +138,56 @@ See [Literals (MS Learn)](https://learn.microsoft.com/en-us/dotnet/fsharp/langua
 
 ## Functions
 
-The `let` keyword also defines named functions.
+Use the `let` keyword to define named functions.
 
     let negate x = x * -1
     let square x = x * x
-    let print x = printfn "The number is: %d" x
+    let print x = printfn $"The number is: {x}"
```

Comment:
is there a `printn` ? the `f` is redundant if you're going down the intrepolation route

Reply:
Interesting. Did not know this.

Reply:
Turns out, no there is not a `printn` or `print`. Just `printf` and `printfn`.

Reply:
IIRC there may be a fslang suggestion/issue for it. Anyway, either way it's obviously just fine as is, sorry!